stix_shifter_utils/modules/cim/stix_translation/json/cim_select_fields.json

```diff
@@ -11,6 +11,18 @@
         "file_hash",
         "user",
         "url",
-        "protocol"
+        "protocol",
+        "host",
+        "source",
+        "DeviceType",
+        "Direction",
+        "severity",
+        "EventID",
+        "EventName",
+        "ss_name",
+        "TacticId",
+        "Tactic",
+        "TechniqueId",
+        "Technique"
     ]
-}
\ No newline at end of file
+}
```

Comment:
same goes to this `cim_select_fields.json` file. make a copy and put that inside splunk module.

Reply:
Resolved